samples/amcache.py

```diff
@@ -213,6 +218,19 @@ def main(argv=None):
         w.writerow(["timestamp", "timestamp_type", "path", "sha1"])
         for e in sorted(entries, key=lambda e: e.timestamp):
             w.writerow([e.timestamp, e.type, e.entry.path, e.entry.sha1])
+
+    elif args.do_json:
+        for e in ee:
+            document = {}
+            for i in FIELDS:
+                document[i.name] = getattr(e, i.name)
+                if document[i.name] is None:
+                    document[i.name] = "-"
+                elif type(document[i.name]) == datetime.datetime:
+                    document[i.name] = str(document[i.name])
```

Comment:
finally, we should probably be explicit in our timestamp formatting, so recommend using:

```
                val = getattr(e, i.name, "-")
                if isinstance(val, datetime.datetime):
                    val = val.isoformat(" ")
                document[i.name] = val
```